test_conformance/c11_atomics/common.h

```diff
@@ -1041,8 +1041,8 @@ CBasicTest<HostAtomicType, HostDataType>::KernelCode(cl_uint maxNumDestItems)
             // global atomics declared in program scope
             code += R"(
                 if(atomic_fetch_add_explicit(&finishedThreads, 1u,
-                                           memory_order_relaxed,
-                                           memory_scope_work_group)
+                                           memory_order_acq_rel,
+                                           memory_scope_device)
```

Comment:
Note #1398 duplicates (and extends) this change; some questions about it support being optional in 3.0 are raised there.